app/models/legislation.rb

```diff
@@ -71,6 +71,10 @@ def to_s(variant = :short)
     )
   end
 
+  def ==(other)
+    other.to_sym == to_sym
```

Comment:
Currently causes failures with other specs – all trying to call `#to_sym` on a `RSpec::Matchers::BuiltIn::HaveAttributes`. Dunno if this isn rspec bug?!

```
  14) Legislation.for_public_body public body tagged as eir_only does not contains FOI legislation
      Failure/Error: other.to_sym == to_sym

      NoMethodError:
        undefined method `to_sym' for #<RSpec::Matchers::BuiltIn::HaveAttributes:0x000055f97c029da8>
        Did you mean?  to_s
      # ./app/models/legislation.rb:75:in `=='
      # ./spec/models/legislation_spec.rb:93:in `block (4 levels) in <top (required)>'
```